ui/app/adt/models/wardDetails.js

```diff
@@ -15,6 +15,14 @@ Bahmni.ADT.WardDetails.create = function (details, diagnosisStatus) {
         return newObject;
     };
 
+    var filterHeadingsFromResponse = function (newObject, oldObject, properties) {
+        var keys = Object.keys(oldObject);
+        var identicalItems = properties.filter(function (item) {
+            return keys.indexOf(item) !== -1;
```

Comment:
This approach is valid but I feel like it can be simplified using the `includes` method, which is more concise and readable. It would be great to have a unit test here as well, to validate this function.